src/services/triggers.service.ts

```diff
@@ -105,11 +100,6 @@ export class TriggersService {
 			const { data: environments, error } = await EnvironmentsService.listByProjectId(projectId);
 
 			if (error) {
-				LoggerService.error(
```

Comment:
don't we need the Logger?

Reply:
Same as above, logged in `EnvironmentsService.listByProjectId`